### ISSUE TITLE
[Legitimate Site Blocked] www.metawash.me

### ISSUE THREAD

**Issue by SumentsJavi** [MetaMask/eth-phishing-detect#5842]
http://www.metawash.me/

Hi. I just bought that domain to put there one of our products.... I am not sure why it is being blocked by your add-on

**Comment by AlexHerman1** [comment]
If there is a plan to host actual content here for a business please open a new issue

**Comment by JavierMoncayo** [comment]
Why should I open a new issue about the same topic? This issue has not been completed and is still presenting the same issue for months...

**Comment by AlexHerman1** [comment]
there is nothing hosted here, if you are hosting something at this site you can request a review.
